src/kms_key_provider.rs

```diff
@@ -23,6 +23,34 @@ impl KMSKeyProvider {
         }
     }
 
+    /// Create a KMS key provider from raw credentials
+    ///
+    /// This method is particularly useful when you can't load credentials from environment variables.
+    pub fn from_credentials(
+        key_id: impl Into<String>,
+        access_key_id: impl Into<String>,
+        secret_access_key: impl Into<String>,
+        session_token: impl Into<String>,
```

Comment:
The session token isn't always necessary; I think this would need to be `Option<Into<String>>` or something of that nature.

Reply:
Thanks, updated 👍🏻